site-content/source/modules/ROOT/pages/development/documentation.adoc

```diff
@@ -8,7 +8,7 @@
 The official Cassandra documentation lives in the project's git
 repository.
 We use a static site generator, http://www.antora.org/[Antora], to create pages hosted at
-{site-url}/doc/latest/[cassandra.apache.org].
+{site-url}doc/latest/[cassandra.apache.org].
```

Comment:
```suggestion
https://cassandra.apache.org/doc/latest/[cassandra.apache.org].
```

Reply:
Mick had requested the removal of all hard coded absolute paths of 'https://cassandra.apache.org'

{site-url} will be processed as https://cassandra.apache.org/'

I can confirm locally that it points to the right place.

Reply:
that's correct – we don't want to hardcode ` https://cassandra.apache.org/` anywhere. (one example: it needs to work the same on  https://cassandra.staged.apache.org/ )

Reply:
Nope, definitely doesn't work. 🙂

`{site-url}` doesn't get rendered at all. This is what it looks like in both [production](https://cassandra.apache.org/_/development/documentation.html) and [staging](https://cassandra.staged.apache.org/_/development/documentation.html):

```
The official Cassandra documentation lives in the project’s git repository. \
  We use a static site generator, Antora, to create pages hosted at //doc/latest/[cassandra.apache.org].
```

I've spent 2 hours last week just on this single issue and couldn't find any other way of making it work. I finally spoke to the Antora experts and they confirmed that since the Doc pages are not asciidoc, Antora doesn't know how to render `{site-url}` and so the link to the docs have to be hardcoded.

Feel free to have a look at http://18.192.24.126:9000/_/development/documentation.html where I have your PR built/staged. Cheers! 🍻

Reply:
> {site-url} doesn't get rendered at all. This is what it looks like in both production and staging:

We are still manually mangling the pages before they hit staging and prod.
See https://github.com/apache/cassandra-builds/compare/trunk...thelastpickle:mck/16765#diff-1f760d89dfb81ea05a11d862007a4e4e586b38aac7ae87f62c5b2e86571809c0R1288-R1290

So `//doc/latest/` will get rewritten to `/doc/latest/`

Reply:
I've updated the revision above to:

```
link:/doc/latest/[cassandra.apache.org].
```